_posts/2019-03-28-improving-geo-replication-status.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  Improving Geo-replication Status
+author: aravindavk
```

Comment:
is there option for tags etc? it may be required in future.

Reply:
tags and categories can be supported, but for now page is not generated to list blog posts of a tag/category.